### PR TITLE
Allow level to be set for each event

### PR DESCRIPTION
**Pull request by kshep**
Instead of specifying a `level` to be used for all events sent to Rollbar, you now specify a `default_level`.

```
  rollbar {
    access_token => "TOKEN"
    endpoint => "https://api-alt.rollbar.com/api/1/item/"
    default_level => "info"
  }
```

You can override this level by adding a field to the event like:

```
filter {
  mutate {
    add_field => { "[rollbar][platform]" => "linux" }
    add_field => { "[rollbar][server][host]" => "%{host}" }
    add_field => { "[rollbar][level]" => "%{level}" }
  }
}
```

Note that if the value passed to Rollbar for level isn't recognized (i.e. isn't one of `debug`, `info`, `warning`, `error`, or `critical`) Rollbar will consider call it an Error even if your default level is something other than Error.

### CODE REVIEW

File: logstash-output-rollbar.gemspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-rollbar'
-  s.version         = '0.1.2'
+  s.version         = '0.1.3'
```
Comment (brianr) [inline logstash-output-rollbar.gemspec]: Seems like a breaking change; how about `0.2.0`?

Comment (kshep) [inline logstash-output-rollbar.gemspec]: Good point. Updated to 0.2.0 in https://github.com/rollbar/logstash-output-rollbar/commit/8bf3b32217ba3ec968e4000e0254aa494286c567

**Comment by goddardcm** [comment]
Was just looking for exactly this behavior. Any reason not to also allow access tokens (and, for that matter, environment) to be set in this same way? I have a logstash configuration that takes in log files from a number of different services and I'd like to be able to dynamically pipe them to the appropriate Rollbar project. 
As is, I would need 21 different outputs configured (3 levels x 7 services). With this change as is, I'm down to 7. Ideally I'd like to get down to 1.

**Comment by kshep** [comment]
@goddardcm That's a great point. I really can't think of any reason not to just have _everything_ work that way. In fact, I'm inclined to change "default_level" back to "level", consider that whole initial config to just be defaults, and let anything be overridden by a specific event.

**Comment by brianr** [comment]
@kshep That seems reasonable to me.

**Comment by kshep** [comment]
@goddardcm This should do it: https://github.com/rollbar/logstash-output-rollbar/commit/a5507d7d4830d7092e4555b07d5b0b6cf4d34ad9

**Comment by goddardcm** [comment]
Looks good to me! Thanks!

**Pull request merged**